Resources/Prototypes/Entities/Markers/Spawners/jobs.yml

```diff
@@ -505,3 +505,15 @@
     layers:
       - state: green
       - state: psychologist
+
+- type: entity
+  id: SpawnPointBlueshield
+  parent: SpawnPointJobBase
+  name: blueshield
+  components:
+  - type: SpawnPoint
+    job_id: Blueshield
+  - type: Sprite
+    layers:
+      - state: green
+      - state: captain
```

Comment:
this should probably have an actual unique state

Reply:
I don't understand how to do